charts/metrics-server/values.yaml

```diff
@@ -131,3 +131,6 @@ tolerations: []
 affinity: {}
 
 topologySpreadConstraints: []
+
+# Annotations to add to the deployment
+annotations: {}
```

Comment:
Would this be ambiguous? Should we use something like `deploymentAnnotations`?

I haven't added this to the values table in README because the table formatting would exaggerate the size of this PR, as happened in https://github.com/kubernetes-sigs/metrics-server/pull/1102#issuecomment-1279543877. I'll wait for it to merge first.

Reply:
```suggestion
deploymentAnnotations: {}
```